apps/token-manager/app/src/components/HolderRow.js

```diff
@@ -83,18 +84,27 @@ class HolderRow extends React.Component {
             </ContextMenuItem>
           </ContextMenu>
         </TableCell>
-      </TableRow>
+      </StyledTableRow>
     )
   }
 }
 
+const StyledTableRow = styled(TableRow)`
+  td:first-child {
```

Comment:
Perhaps we could change the first `TableCell` instead of adding a style for the entire row and then using css to select the first cell?

Reply:
I agree yet I'm sure that's where I started and was not able to override the style and had to do this - I'll double check.

Reply:
Updated :)